src/Controls/samples/Controls.Sample.UITests/Issues/Issue20439.xaml.cs

```diff
@@ -3,8 +3,26 @@
 
 namespace Maui.Controls.Sample.Issues
 {
-	[XamlCompilation(XamlCompilationOptions.Compile)]
 	[Issue(IssueTracker.Github, 20439, "[iOS] Double dash in Entry or Editor crashes the app", PlatformAffected.iOS)]
+	public class Issue20439Test : ContentPage
```

Comment:
The runners don't really work with Shell right now so I changed this one to swap in the main page as part of the test